workshops/py_Christmas/README.md

```diff
@@ -0,0 +1,380 @@
+        turtle.left(90)
+```
+The code above is the one that draws the rectangle. This code moves the turtle to one side by the same distance as the width mentioned. Then, turns it to left (left of the turtle) and moves it to a distance same as the height. This process is repeated twice as it is put in a loop twice. Now we have a complete rectangle. 
+Are you still with me? 
+
+Let's make the star now shall we? Check the code below and see if you understand.
+
+```python
+def star(turtle, color, x, y, size):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+```
+I guess you would've understood atleast 90% of the above code easily. A brief description is given below.
+The ******size****** parameter decides the length of the edges of the star.
+Here, we move the turtle forward by a distance same as that of the size mentioned. Now, we turn the turtle to right by 144 degrees and move it by the same distance. Repeat the above steps 5 times and you will have a star. Got it? 
+
+Last function in this file to make the tiny balls on either sides of the boxes (green part) of the Christmas tree.
+Go through the code. You will probably understand it pretty easily.
+```python
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+``` 
+Yes, ******turtle****** library has a circle function. Made our job pretty easy, didn't it? 
+
+ Our functions to make the figures is ready. Let's take a look at the whole code.
+
+```python
+import turtle
+def box1 (turtle, color, x, y, width, height):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x,y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range(2):
+        turtle.forward(width)
+        turtle.left(90)
+        turtle.forward(height)
+        turtle.left(90)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+
+def star(turtle, color, x, y, size):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    for i in range (5):
+        turtle.forward(size)
+        turtle.right(144)
+        turtle.forward(size)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+
+
+def balls(turtle, color, x, y, radius):
+    turtle.penup()
+    turtle.color(color)
+    turtle.fillcolor(color)
+    turtle.goto(x, y)
+    turtle.pendown()
+    turtle.begin_fill()
+    turtle.circle(radius)
+
+    turtle.end_fill()
+    turtle.setheading(0)
+``` 
+All good? 
+
+# Now, lets move on to the next file : main.py
```

Comment:
Also, the length of header text should generally be pretty short, not a full sentence. Maybe call this header “The main.py file” and make the current sentence the first line under the header. For example:

# The main.py file

Now, let’s move on to the next file: main.py. Open the main.py file by clicking on it on your sidebar.